ios/RN/RNCamera.m

```diff
@@ -254,6 +254,10 @@ - (void)updateFlashMode
 {
     AVCaptureDevice *device = [self.videoCaptureDeviceInput device];
     NSError *error = nil;
+    
+    if(device == nil){
```

Comment:
why so many decide checks?

Reply:
They don't really change much if they are there or not, because the following lines will always end up exiting due to how objective C handles nil (there won't be a null pointer exception), but since most props are set by RN (all setters called) when the component mounts but the camera is not ready, all those setters will end up failing / doing nothing. Those nil checks will just return early and avoid running nil through all the following calls.